.circleci/config.yml

```diff
@@ -3,24 +3,28 @@ version: 2.1
 orbs:
   cypress: cypress-io/cypress@1
 
-
 workflows:
   build:
     jobs:
-      - build_and_test
-
-      - cypress_install
+      - build_and_test:
+          requires:
+            - cypress/run
 
       - cypress/run:
           requires:
             - cypress_install
           record: true # record results on Cypress Dashboard
-          parallel: true # split all specs across machines
```

Comment:
Why did we get rid of parallel tests?

Reply:
I suspected maybe this had something to do with Cypress tests failing. But I know it shouldn't have anything to do with it and really I was just playing around with the configuration and also trying to make the merging of cypress and jest test results easier for me in the CI process.

Currently, working on going back to having parallel tests.